src/Tizen.NUI/src/public/AccessibilityManagerEventArgs.cs

```diff
@@ -0,0 +1,835 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI
+{
+    public partial class AccessibilityManager
+    {
+        /// <summary>
+        /// Event arguments that passed via StatusChanged signal
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class StatusChangedEventArgs : EventArgs
+        {
+            private AccessibilityManager _accessibilityManager;
+
+            /// <since_tizen> 6 </since_tizen>
+            /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public AccessibilityManager AccessibilityManager
```

Comment:
this seems to be an old code pattern. in EventHandler, the first argument is object which is sender. (in this case, AccessibilityManager) so you don't need to pass AccessibilityManger instance to event subscriber through the EventArgument. This pattern is old style, and it is seen in may places. please remove this.

Reply:
Okay, Removed the old coding style and changed to the latest one. (similar to other classes)
Please  check it again.